peekingduck/cli.py

```diff
@@ -66,10 +66,11 @@ def create_yml() -> None:
 
 
 @cli.command()
-def init() -> None:
+@click.option('--custom_folder_name', default='custom_nodes')
```

Comment:
clarifying something here, if `--custom_folder_name` is not specified will `custom_folder_name=None`?

Reply:
If `--custom_folder_name` is not defined, then it will use the default, which is `custom_nodes`